### PR TITLE
fix: HTTP status codes fixed

### LINKED ISSUE #923: Use HTTP Status Codes instead of hard coded status codes
**Describe the bug**

Some routes have hard coded status codes instead of HTTP Status 
**To Reproduce**

1. Go to https://mentorship-backend-temp.herokuapp.com/
2. Click on `User`
3. Scroll down to /user (Returns details of current user)

**Expected behavior**
HTTPStatus.XYZ should have been displayed instad of raw status codes

**Screenshots**
error_code

**Condition for Done**

- [ ]  All hard coded status codes are removed

**Estimated Time**
1 hour

### PR DESCRIPTION
**Pull request by 17sushmita**
### Description

HTTP status codes added to all the endpoints and removed hardcoded status codes
Fixes #923

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/api/resources/user.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ class VerifiedUser(Resource):
             f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
         }
     )
-    @users_ns.marshal_list_with(public_user_api_model)  # , skip_none=True
+    @users_ns.marshal_list_with(public_user_api_model, code=HTTPStatus.OK, description="Success")  # , skip_none=True
```
Comment (devkapilbansal) [inline app/api/resources/user.py:227]: @17sushmita why you add status code here? If not done purposely please revert

**Review by devkapilbansal — CHANGES_REQUESTED** [review]

File: app/api/resources/user.py (hunk 1 above)
Comment (devkapilbansal) [reply app/api/resources/user.py:227]: Same for other instances as well

Comment (17sushmita) [reply app/api/resources/user.py:227]: I've done it purposely as not mentioning status code in marshal statement was the reason why hardcoded status code was displayed in response table. This statement was automatically generating a response example and taking 200 as status code implicitly.

Comment (devkapilbansal) [reply app/api/resources/user.py:227]: Not sure about this. Pinging @PrashanthPuneriya for review

Comment (devkapilbansal) [reply app/api/resources/user.py:227]: Thanks @17sushmita for your explanation

**Review by devkapilbansal — DISMISSED** [review]
Thanks @17sushmita

LGTM :+1: 

The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

  * _Test1 Description_  Hard coding of HTTP Status Code
    _Screenshot/gif_:  
**From develop branch**
error_code
error_2

**From this PR**
http_status
admin_success

    _Expected Result_:  There should be no instances were the status code is hard-coded.
    _Actual Result_: Actual Result is same as expected result
  
3 . Additional Testcases Covered: Ran `python -m unittest discover tests` to ensure all test cases passed.

4 . Additional Comments: - Manually verified for all routes if the status code is hard coded or not.

**Comment by devkapilbansal** [comment]
@PrashanthPuneriya @vj-codes can you please add your review

**Comment by 17sushmita** [comment]
I've squashed the commits.

**Review by PrashanthPuneriya — APPROVED** [review]
LGTM!! Thanks for your contribution :)

**Review by devkapilbansal — APPROVED** [review]

File: app/api/resources/user.py (hunk 1 above)
Comment (PrashanthPuneriya) [reply app/api/resources/user.py:227]: You have done a great job in figuring this out @17sushmita :)

**Comment by vj-codes** [comment]
Code review ✔️ 
Adding label ready to merge since testing is done above

**Review by isabelcosta — APPROVED** [review]
Amazing work @17sushmita 👏 🎉 
Thank you so much for your contribution to mentorship-backend 🤗

**Pull request merged**